product_dimension_compute_volume/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# SPDX-FileCopyrightText: 2022 Coop IT Easy SCRLfs
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+{
+    "name": "Product Dimension Compute Volume",
+    "summary": """
+        Automatically compute the volume depending on the dimensions.""",
+    "version": "12.0.1.0.0",
+    "category": "Product",
+    "website": "https://coopiteasy.be",
+    "author": "Odoo Community Association (OCA), Coop IT Easy SCRLfs",
+    "maintainer": "Coop IT Easy SCRLfs",
+    "maintainers": ["coopiteasy"],
```

Comment:
Setting organisation doesn't work for the time being. Set a github user instead. (Carmenbianca)

Reply:
Will fix this.